scipy/optimize/_linprog_highs.py

```diff
@@ -330,5 +345,10 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
            'message': statuses[res['status']][1],
            'nit': res.get('simplex_nit', 0) or res.get('ipm_nit', 0),
            'crossover_nit': res.get('crossover_nit'),
+           'method': solvers[solver],
```

Comment:
Can `_highs_wrapper` return the solver used?

Reply:
No. As HiGHS develops, there will be a dominant solver algorithm in many cases, but in others it won't be clear, as the strategy of the "chosen" solution technique becomes more sophisticated. The interested user can look at the solution log. I know of no other solver that returns such information.

Reply:
ok